src/core/server/integration_tests/saved_objects/routes/find.test.ts

```diff
@@ -67,6 +83,33 @@ describe('GET /api/saved_objects/_find', () => {
     );
   });
 
+  it('returns with status 400 when type is hidden from the HTTP APIs', async () => {
+    const findResponse = {
+      error: 'Bad Request',
+      message: 'Unsupported saved object type(s): hidden-from-http: Bad Request',
+      statusCode: 400,
+    };
+    const result = await supertest(httpSetup.server.listener)
+      .get('/api/saved_objects/_find?type=hidden-from-http')
+      .expect(400);
```

Comment:
This change in the response is new. Before, `find` would (always?) return 200. Now the response depends on the type we're tying to find.

Reply:
Yeah, it's slightly awkward to have a difference of behavior between `hidden` (filtered out) and `hiddenFromHTTPApis` (throw), but it's probably fine. Plus, ihmo the correct behavior is throwing, it's just that we can't really change the existing behavior without it being a breaking change...

Reply:
> we can't really change the existing behavior without it being a breaking change...

Do we need to raise the change with the BReaking Change Committee or is it ok to add the change to the release notes only?

Reply:
Oh, I think I misread your comment regarding the breaking change. IIUC, right now the difference in `find` response isn't a breaking change since we don't have any types registered with the new option enabled. If and when saved object owners do make that change, they would have to handle the changed response into account.